tests/testingData/files.json

```diff
@@ -117,13 +117,28 @@
   "createEvent": {
     "title": "title1 is good",
     "description": "Bomaye",
-    "body": "big booty",
+    "body": "event body",
```

Comment:
Hey @Kasulejoseph, just noticed that the event body and description are likely the same things, i think we should keep one of them.accross all events.